cmd/activator/main.go

```diff
@@ -242,19 +242,21 @@ func main() {
 		logger.Fatalw("Unable to create request log handler", zap.Error(err))
 	}
 	ah = reqLogHandler
-	ah = &activatorhandler.ProbeHandler{NextHandler: ah}
 
-	sigCtx, sigCancel := context.WithCancel(context.Background())
+	// NOTE: MetricHandler is being used as the outermost handler of the meaty bits. We're not interested in measuring
+	// the healthchecks or probes.
+	ah = activatorhandler.NewMetricHandler(ctx, reporter, ah)
+	ah = activatorhandler.NewContextHandler(ctx, ah)
```

Comment:
This does invoke lister and what not, may be include it in the metric handler?

Reply:
I specifically did not do that to not require specific ordering of the MetricHandler.